packages/shared/providers.yaml

```diff
@@ -492,6 +492,15 @@ chorus:
     proxy:
         base_url: https://chorus.ai
     docs: https://docs.nango.dev/integrations/all/chorus
+circle:
+    categories:
+        - communication
+    auth_mode: API_KEY
+    proxy:
```

Comment:
`proxy.verification` won't work here as the provider always returns `status: 200`. I had to manually test with a small sync script.

Reply:
What does it return if it fails? An error in the payload? Anything in the headers?

Reply:
@khaliqgant , the response returns with a status of `200` even after sending the wrong APIkey. A good example is: `{ "status": "unauthorized", "message": "Your account could not be authenticated." }`